elizhahn/karaoke-stretch
# Feature/add song

## What does this PR do (summary of changes)?
- An addSong method was added to the App component 
 - This. method was created for the /songbook route and will allow a user to add a song to their personal collection
## How should it be tested?
 - fetch the branch and npm start
 - navigate to song book and add a one of the rendered songs, then navigate to my songs and you should see the card rendered in that route
## Future Iterations:
 - a remove song method will be added
 - loading features will be added
 - classes will be passed to hide the button once added
